app.js

```diff
@@ -136,17 +135,16 @@ if (Config.crashguard) {
 
 global.Sockets = require('./sockets');
 
-exports.listen = function (port, bindAddress, workerCount) {
-	Sockets.listen(port, bindAddress, workerCount);
+exports.listen = function (port) {
```

Comment:
Would making it possible to pass `bindAddress`, `workerCount` through `process.env.argv` like you can with `port` be a good idea? I don't see much point in them being parametres otherwise